content/_partials/downloadbanner.html.haml

```diff
@@ -7,7 +7,7 @@
       .col-md-1.col-lg-2
 
       .col-md-5.col-lg-4
-        %img{:src => "/images/logos/jenkins/jenkins.svg", :style => 'width: 256px;'}
+        %img{:src => "/images/logos/jenkins/STOP_the_WAR.jpeg", :style => 'width: 400px;'}
```

Comment:
is there an SVG or PNG we can use instead? 

The JPEG has a white background that is visible on the homepage of the preview site?

Reply:
We're confident that we can convert the JPEG to a PNG with transparent background or to an SVG.  Wanted to make the prototype visible before we did the image format changes to have a transparent background.